signup-default.properties

```diff
@@ -0,0 +1,93 @@
+
+spring.messages.basename=messages
+spring.messages.encoding=UTF-8
+
+spring.main.allow-bean-definition-overriding=true
+
+spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration, \
+  org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration, \
+  org.springframework.boot.autoconfigure.jdbc.DataSourceTransactionManagerAutoConfiguration
+
+## ---------------------------------------- Cache configuration --------------------------------------------------------
+
+spring.cache.type=simple
+
+#spring.cache.type=redis
+#spring.cache.cache-names=${mosip.esignet.cache.names}
+#spring.redis.host=localhost
+#spring.redis.port=6379
+
+mosip.esignet.cache.names=challenge-generated,challenge-verified,status-check
+mosip.esignet.cache.size={'challenge-generated': 200,'challenge-verified': 200,'status-check': 200 }
+mosip.esignet.cache.expire-in-seconds={'challenge-generated': 86400,'challenge-verified': 86400,'status-check': 86400 }
+
+## ------------------------------------- Auth adapter ------------------------------------------------------------------
+mosip.kernel.authmanager.url=https://api-internal.camdgc-dev.mosip.net
+keycloak.external.url=https://iam.camdgc-dev.mosip.net
+keycloak.internal.url=https://iam.camdgc-dev.mosip.net
+auth.server.validate.url=${mosip.kernel.authmanager.url}/v1/authmanager/authorize/admin/validateToken
+auth.server.admin.issuer.uri=${keycloak.external.url}/auth/realms/
+auth-token-generator.rest.issuerUrl=${keycloak.internal.url}/auth/realms/mosip
+mosip.keycloak.issuerUrl=${keycloak.internal.url}/auth/realms/mosip
+mosip.auth.adapter.impl.basepackage=io.mosip.kernel.auth.defaultadapter
+mosip.kernel.auth.adapter.ssl-bypass=true
+mosip.kernel.auth.appid-realm-map={admin:'mosip',crereq:'mosip',creser:'mosip',idrepo:'mosip', signup:'mosip'}
+mosip.kernel.auth.appids.realm.map={admin:'mosip',crereq:'mosip',creser:'mosip',idrepo:'mosip','regproc':'mosip', signup:'mosip'}
+
+mosip.iam.adapter.clientid=mosip-signup-client
+mosip.iam.adapter.clientsecret=QdDb3zAdFDgBvto7aNasLLJjxZ1xSSvu
+mosip.iam.adapter.appid=signup
+mosip.iam.adapter.issuerURL=${keycloak.external.url}/auth/realms/mosip
+mosip.authmanager.client-token-endpoint=${mosip.kernel.authmanager.url}/v1/authmanager/authenticate/clientidsecretkey
+mosip.iam.adapter.validate-expiry-check-rate=15
+mosip.iam.adapter.renewal-before-expiry-interval=15
+mosip.iam.adapter.self-token-renewal-enable=true
+
+mosip.service-context=${server.servlet.context-path}
+mosip.service.end-points=/**/*
+mosip.service.exclude.auth.allowed.method=GET,POST
+## -------------------------------------------- Sign Up  ----------------------------------------------------------
+mosip.signup.ui.config.key-values={\
+ 'identifier.pattern': '\\d{8,9}', \
+'identifier.prefix': '+855', \
+'captcha.site.key': '', \
+ 'otp.length': 6, \
+'password.pattern': '^[a-zA-Z0-9]{8,20}$', \
+'challenge.timeout': 60, \
+ 'resend.attempts': 3, \
```

Comment:
value of resend.attempts should be placeholder ${mosip.signup.challenge.resend-attempt}